Win11ThemeTest/ButtonTest.cs

```diff
@@ -0,0 +1,126 @@
+﻿using FlaUI.Core;
+using FlaUI.Core.AutomationElements;
+using FlaUI.Core.Input;
+using FlaUI.UIA3;
+using FlaUI.Core;
+using FlaUI.Core.AutomationElements;
+using FlaUI.Core.Conditions;
+using FlaUI.Core.Definitions;
+using FlaUI.Core.Input;
+using FlaUI.UIA3;
```

Comment:
some namespaces are repetitive, clean up the code, remove unnecessary usings, white spaces, comments from all the tests

Reply:
The repetitive namespaces and additional whitespaces are removed.